dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/business/LanguageCacheImpl.java

```diff
@@ -14,7 +14,11 @@
  */
 public class LanguageCacheImpl extends LanguageCache {
 
-	private static final String ALL_LANGUAGES_KEY="ALL_LANGUAGES_KEY";
+	static final String ALL_LANGUAGES_KEY="ALL_LANGUAGES_KEY";
+	public static Language LANG_404 = new Language(-1,
+			"LANG__404", "LANG__404", "LANG__404",
```

Comment:
Codacy found an issue: [The String literal "LANG__404" appears 4 times in this file; the first occurrence is on line 19](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5709025)